src/Organizations/OrganizationForm/OrganizationEDIInfoForm/OrganizationEDIInfoForm.js

```diff
@@ -190,7 +192,7 @@ class OrganizationEDIInfoForm extends Component {
                     label={<FormattedMessage id="ui-organizations.edi.serverAddress" />}
                     name="edi.ediFtp.serverAddress"
                     type="text"
-                    validate={[isURLValid]}
+                    validate={[validateURL]}
```

Comment:
in case of change I would remove array here, so it's easy to switch to final-form later